pkg/network/networknamescheme/networknamescheme.go

```diff
@@ -19,4 +19,35 @@
 
 package networknamescheme
 
+import (
+	"fmt"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
 const DefaultNetworkName = "eth0"
+
+func GetNetworkNameSchemeMap(vmi *v1.VirtualMachineInstance) map[string]string {
+	networkNameSchemeMap := map[string]string{}
+
+	mapDefaultNetworkToPodInterfaceName(vmi, networkNameSchemeMap)
+	mapMultusNonDefaultNetworksToPodInterfaceName(vmi, networkNameSchemeMap)
+	return networkNameSchemeMap
+}
+
+func mapMultusNonDefaultNetworksToPodInterfaceName(vmi *v1.VirtualMachineInstance, networkNameSchemeMap map[string]string) {
```

Comment:
I think you can create and return an initialized and populated map instead of mutating an existing one.
Whenever we mutate, we should feel bad about it :)

Reply:
no mutatos.

Reply:
I agree :)